crawler/crawler/crawler.go

```diff
@@ -303,19 +303,20 @@ func (c *Crawler) ProcessRepo(repository Repository) {
 	}
 }
 
-func validateRemoteFile(data []byte, fileRawURL string, pa PA) error {
-	parser, err := getRemoteFile(data, fileRawURL, pa)
+func validateRemoteFile(data []byte, fileRawURL string, pa PA, domain Domain) error {
+	parser, err := getRemoteFile(data, fileRawURL, pa, domain)
 	if err != nil {
 		return err
 	}
 	return validateFile(pa, parser, fileRawURL)
 }
 
-func getRemoteFile(data []byte, fileRawURL string, pa PA) (publiccode.Parser, error) {
+func getRemoteFile(data []byte, fileRawURL string, pa PA, domain Domain) (publiccode.Parser, error) {
 	parser := publiccode.NewParser()
 	parser.Strict = false
 	parser.RemoteBaseURL = strings.TrimRight(fileRawURL, viper.GetString("CRAWLED_FILENAME"))
 
+	// err := parser.ParseInDomain(data, domain.Host, domain.BasicAuth)
```

Comment:
I don't get it. You're passing domain to the getRemoteFile func but not using it in the parser?

Reply:
good catch!

Reply:
right call to parser must be line :319